packages/progress-indicator/src/LionProgressIndicator.js

```diff
@@ -78,12 +125,45 @@ export class LionProgressIndicator extends LocalizeMixin(LitElement) {
 
   connectedCallback() {
     super.connectedCallback();
-    this.setAttribute('role', 'status');
-    this.setAttribute('aria-live', 'polite');
+    this.setAttribute('role', 'progressbar');
+  }
+
+  /**
+   * Update aria labels on state change.
+   * @param {import('@lion/core').PropertyValues } changedProperties
+   */
+  updated(changedProperties) {
+    if (changedProperties.has('value') && !this.indeterminate) {
```

Comment:
You forgot to call `super.updated(changedProperties);`  ;)

Could we add at the top: 
```js
if (this.indeterminate) {
  return;
}
```

(and then drop all `!this.indeterminate` checks below)